security.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Security
+description: "Learn more about the Fiddler Everywhere security highlights and check out the related technical information about sharing and capturing functionalities."
+slug: fe-security
+publish: true
+position: 75
+---
+
+# Fiddler Everywhere Security
+
+The Fiddler Everywhere application acts as a MITM proxy that uses its unique certificate. It will capture incoming and ongoing internet activities, and you need to be careful not to share any sensitive data as it might pose a threat to you or your company. This article highlights the primary security concerns and provides technical information about how Fiddler Everywhere acts as a MITM and what you should consider when handling the Fiddler root trust certificate and captured data.
+
+
+## How Does Fiddler Everywhere Work
+
+Fiddler Everywhere is an HTTP/HTTPS proxy that runs on your Windows, Linux, or Mac computer. It stands between your client application (for example, a browser or other application that sends HTTP/HTTPS requests) and the server (that receives the HTTP/HTTPS requests). The HTTP/HTTPS responses are returned to the Fiddler Everywhere proxy, passing the traffic back to the client application.
```

Comment:
`Fiddler Everywhere is an HTTP/HTTPS proxy that runs on your Windows, Linux, or Mac computer.` - Windows, Linux and macOS - those are operating systems. The correct name is `macOS`